src/core/server/graph/schema/schema.graphql

```diff
@@ -2429,6 +2442,7 @@ type User {
   ssoURL is the url for managing sso account
   """
   ssoURL: String
+    @auth(userIDField: "id", permit: [SUSPENDED, BANNED, PENDING_DELETION])
```

Comment:
is this related or just a fix while you're here?

Reply:
Yes. It's only accessed via the `viewer` in our codebase, and shouldn't be hit unless the user is logged in, which this `@auth` directive is just enforcing.